examples/turnstile/trdelnik-tests/tests/test.rs

```diff
@@ -84,9 +84,17 @@ impl Fixture {
         self.client
             .airdrop(self.client.payer().pubkey(), 5_000_000_000)
             .await?;
+
         self.client
             .deploy(self.program.clone(), mem::take(&mut program_data))
             .await?;
+
+        println!(
```

Comment:
Is it possible to replace the `println!` with `assert`? I think people will be using this method to check the balance before and after submitting a transaction. So we can check if payer's balance is `5_000_000_000` for example.

Reply:
Done :heavy_check_mark: